cmd/nerdctl/network_inspect_test.go

```diff
@@ -66,3 +66,24 @@ func TestNetworkInspect(t *testing.T) {
 	}
 	assert.DeepEqual(base.T, expectedIPAM, got.IPAM)
 }
+
+func TestNetworkInspectWithNamespace(t *testing.T) {
```

Comment:
Probably this can be safely merged to `TestNetworkLsWithNamespace`, but no strong opinion

Reply:
Good idea. I merged them all actually - much more compact.